src/app/forms/templates/small-trailer/small-trailer-tech-record.template.ts

```diff
@@ -65,12 +65,10 @@ export const SmallTrailerTechRecord: FormNode = {
     {
       name: 'euVehicleCategory',
       label: 'EU vehicle category',
-      value: '',
+      value: EuVehicleCategories.O1,
       type: FormNodeTypes.CONTROL,
-      editType: FormNodeEditTypes.SELECT,
-      width: FormNodeWidth.S,
-      options: getOptionsFromEnum(EuVehicleCategories),
-      validators: []
+      editType: FormNodeEditTypes.HIDDEN,
```

Comment:
Maybe disabling it would be better than hiding it? Not sure

Reply:
Suggestion applied.